pkg/plugins/plugins.go

```diff
@@ -73,33 +77,40 @@ func (pm *PluginManager) Init() error {
 	}
 
 	pm.log.Info("Starting plugin search")
+
+	pm.log.Info("Scanning main plugin directory")
```

Comment:
Move `plugDir := path.Join(setting.StaticRootPath, "app/plugins")` befire this and include in log?

```suggestion
	pm.log.debug("Scanning core plugin directory", "dir", pluginDir)
```